README.md

```diff
@@ -1,2 +1,160 @@
+           },
+           {
+             "key": "customerId",
+             "value": "ca32853c-48fa-40be-ae75-77b37504581b"
+           }
+         ]
+       }
+     }' \
+     "http://127.0.0.1:8080/1.0/kb/accounts/<ACCOUNT_ID>/paymentMethods?isDefault=true"
+```
+
+Notes:
+* Make sure to replace *ACCOUNT_ID* with the id of the Kill Bill account
+* Details for working payment methods are available here: https://docsv2.dwolla.com/#funding-sources
+
+To trigger a payment:
+
+```
+curl -v \
+     -u admin:password \
+     -H "X-Killbill-ApiKey: bob" \
+     -H "X-Killbill-ApiSecret: lazar" \
+     -H "Content-Type: application/json" \
+     -H "X-Killbill-CreatedBy: demo" \
+     -X POST \
+     --data-binary '{"transactionType":"PURCHASE","amount":"5","currency":"EUR","transactionExternalKey":"INV-'$(uuidgen)'-PURCHASE"}' \
+    "http://127.0.0.1:8080/1.0/kb/accounts/<ACCOUNT_ID>/payments"
+```
+
+Notes:
+* Make sure to replace *ACCOUNT_ID* with the id of the Kill Bill account
+
+At this point, the payment will be in *PENDING* state, until we receive a notification from Dwolla. You can verify the state of the transaction by listing the payments:
+
+```
+curl -v \
+     -u admin:password \
+     -H "X-Killbill-ApiKey: bob" \
+     -H "X-Killbill-ApiSecret: lazar" \
+     -H "Content-Type: application/json" \
+     -H "X-Killbill-CreatedBy: demo" \
+    "http://127.0.0.1:8080/1.0/kb/accounts/<ACCOUNT_ID>/payments?withPluginInfo=true"
+```
+
+You can simulate a notification from Dwolla as follows:
+
+```
+curl -v \
+     -u admin:password \
+     -H "X-Killbill-ApiKey: bob" \
+     -H "X-Killbill-ApiSecret: lazar" \
+     -H "Content-Type: application/json" \
+     -H "X-Killbill-CreatedBy: demo" \
+     -X POST \
+     --data-binary '
+     {
+      "id": "80d8ff7d-7e5a-4975-ade8-9e97306d6c16",
+      "topic": "customer_transfer_completed",
+      "_links": {
+        "self": {
+          "href": "https://api-uat.dwolla.com/events/80d8ff7d-7e5a-4975-ade8-9e97306d6c16"
+        },
+        "account": {
+          "href": "https://api-uat.dwolla.com/accounts/b3153e29-b25a-4ccd-9c1a-5b05b606e50c"
+        },
+        "resource": {
+          "href": "https://api-uat.dwolla.com/transfers/7c8e7484-2184-e611-80e8-0aa34a9b2388"
+        },
+        "customer": {
+          "href": "https://api-uat.dwolla.com/customers/b3153e29-b25a-4ccd-9c1a-5b05b606e50c"
+        }
+      }
+    }' \
+    "http://127.0.0.1:8080/1.0/kb/paymentGateways/notification/killbill-dwolla"
+```
+
+Notes:
+* Make sure to replace *resource.href* with the Dwolla transfer id reference of your payment (see the *dwolla_responses* table)
+* If *topic* is `customer_transfer_completed`, the payment transaction state will be *SUCCESS* and the payment state *PROCESSED*
+* If *topic* is `customer_transfer_failed` or `customer_transfer_canceled`, the payment transaction state will be *PAYMENT_FAILURE* and the payment state *ERROR* or *CANCELED*
```

Comment:
👍 on the docs.

Nit: if `CANCELED`, the Kill Bill state will be `PLUGIN_FAILURE`.